app/features/notifications/actions.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import { SHOW_NOTIFICATION, HIDE_NOTIFICATION } from './actionTypes';
+
+/**
+ * Removes the notification with the passed in id.
+ *
+ * @param {string} uuid - The unique identifier for the notification to be
+ * removed.
+ * @returns {{
+ *     type: HIDE_NOTIFICATION,
+ *     uuid: number
+ * }}
+ */
+export function hideNotification(uuid: number) {
+    return {
+        type: HIDE_NOTIFICATION,
+        uuid
+    };
+}
+
+/**
+ * Queues a notification for display.
+ *
+ * @param {Object} notification - The notification details.
+ * @returns {{
+ *     type: SHOW_NOTIFICATION,
+ *     notification: Object,
+ *     uuid: number
+ * }}
+ */
+export function showNotification(notification: Object = {}) {
+    return {
+        type: SHOW_NOTIFICATION,
+        notification,
+        uuid: window.Date.now()
```

Comment:
can you use the node-uuid package instead and create a uuid v4 please?

Reply:
Done.